triplets/x64-win-llvm/x64-win-llvm.toolchain.cmake

```
@@ -50,7 +50,10 @@ set(ignore_werror "/WX-")
 cmake_language(DEFER CALL add_compile_options "/WX-") # make sure the flag is added at the end!
 
 # general architecture flags
-set(arch_flags "-mcrc32")
+set(arch_flags "")
+if(NOT VCPKG_TARGET_ARCHITECTURE STREQUAL arm64)
+  string(APPEND arch_flags "-mcrc32")
+endif()
```

```suggestion
if(VCPKG_TARGET_ARCHITECTURE STREQUAL x64)
  string(APPEND arch_flags "-mcrc32")
endif()
```

mcrc32 is for x86/x86_64 specific instructions.

---

Fixed. BTW, it is very wired that this was working before.